bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/database/ShadeCapabilitiesDatabase.java

```diff
@@ -79,9 +79,9 @@ public class ShadeCapabilitiesDatabase {
             new Type(47).capabilities(7).text("Pleated Top Down Bottom Up"),
             new Type(49).capabilities(0).text("AC Roller"),
             new Type(51).capabilities(2).text("Venetian"),
-            new Type(54).capabilities(3).text("Vertical Slats Left Stack"),
-            new Type(55).capabilities(3).text("Vertical Slats Right Stack"),
-            new Type(56).capabilities(3).text("Vertical Slats Split Stack"),
+            new Type(54).capabilities(4).text("Vertical Slats Left Stack").capabilitiesOverride(3),
+            new Type(55).capabilities(4).text("Vertical Slats Right Stack").capabilitiesOverride(3),
+            new Type(56).capabilities(4).text("Vertical Slats Split Stack").capabilitiesOverride(3),
```

Comment:
I'm now wondering: From which source did we get the specification of capabilities 3 and 4? Capabilities 4 is not referenced anymore, so the problem could in theory also be wrong definitions. For example, adding `tiltAnywhere()` and `tilt180()` to capabilities 4 (so it would be identical to capabilities 3) would also solve the problem (again, theoretically).

Reply:
Agreed. However I would like to try, if possible, to harmonise the OH approach with that of HA, in order to avoid confusion.

> Capabilities 4 is not referenced anymore

Actually shade types 69,70,71 do reference capabilities:4, and from the description 'curtain' I would assume that they do not support tilt. Which is consistent with what capabilities:4 currently says. However I think that neither @kingy444 nor we have actually seen a real user with any of those types. However if / when such a user does appear then we could confirm or deny the suppositions.

Reply:
You are right, I missed that.